### ISSUE TITLE
Add link to grafana in notification

### ISSUE THREAD

**Issue by korthout** [zeebe-io/zeebe-cluster-testbench#89]
The current Slack notification already contains a link to the operate of the failed test. For quicker investigation it would be great to have a link to the corresponding grafana dashboard as well

**Comment by pihme** [comment]
@korthout Good idea! Do you want to implement this? Would be a good opportunity to get familiar with the testbench

**Comment by korthout** [comment]
Sure, I'll give it a try

**Comment by pihme** [comment]
I can be your sparring partner and review buddy. Let me know when you have any questions.

One detail worth noting: as part of #100 I am working on an option to deploy it to the dev or int environment. (which correlated to Camunda Cloud Dev and Camunda Cloud Int)

So the link should ideally work in both environments.

**Comment by pihme** [comment]
Implementation Hints:
* Extend GatherInformationAboutClusterInCamundaCloudHandler to make the information available in the workflow
* Extend RecordTestResultHandler to write it into the Google Sheet
* Extend NotifyEngineersHandler to write it into the Slack post

The most minimal implementation would be to just change `NotifyEngineersHandler`. For future proofing I would prefer the more elaborate way described above, though. (I am still dreaming of a day when parts of the testbench will use the Grafana link to crawl through the lmetrics and find out what went wrong)

In any case, please remember to also update the service task reference documemntation.
